The test HealthCheckTest.testBookKeeperDown always failed

The test HealthCheckTest.testBookKeeperDown failed in a lot of PRs.

Failed log:

```
2020-12-02T11:54:09.2481018Z [ERROR] Tests run: 20, Failures: 1, Errors: 0, Skipped: 1, Time elapsed: 937.642 s <<< FAILURE! - in TestSuite
2020-12-02T11:54:09.2482492Z [ERROR] testBookKeeperDown(org.apache.pulsar.tests.integration.cli.HealthCheckTest)  Time elapsed: 0.201 s  <<< FAILURE!
2020-12-02T11:54:09.2485899Z org.apache.pulsar.tests.integration.docker.ContainerExecException: pkill -STOP -f BookieServer failed on d1818c9341e5e0a613a675cf6e71bfa2498c9fa4a707da03c8430ad4e456cfbe with error code 1
2020-12-02T11:54:09.2488541Z 	at org.apache.pulsar.tests.integration.utils.DockerUtils$2.onComplete(DockerUtils.java:259)
2020-12-02T11:54:09.2490717Z 	at com.github.dockerjava.core.exec.AbstrAsyncDockerCmdExec$1.onComplete(AbstrAsyncDockerCmdExec.java:51)
2020-12-02T11:54:09.2494340Z 	at com.github.dockerjava.okhttp.OkHttpInvocationBuilder.lambda$executeAndStream$4(OkHttpInvocationBuilder.java:331)
2020-12-02T11:54:09.2496456Z 	at java.lang.Thread.run(Thread.java:748)
2020-12-02T11:54:09.2496864Z 
2020-12-02T11:54:09.5358659Z [INFO] 
2020-12-02T11:54:09.5359070Z [INFO] Results:
2020-12-02T11:54:09.5361212Z [INFO] 
2020-12-02T11:54:09.5362034Z [ERROR] Failures: 
2020-12-02T11:54:09.5366481Z [ERROR] org.apache.pulsar.tests.integration.cli.HealthCheckTest.testBookKeeperDown(org.apache.pulsar.tests.integration.cli.HealthCheckTest)
2020-12-02T11:54:09.5370008Z [INFO]   Run 1: PASS
2020-12-02T11:54:09.5377712Z [ERROR]   Run 2: HealthCheckTest.testBookKeeperDown » ContainerExec pkill -STOP -f BookieServer...
2020-12-02T11:54:09.5379060Z [INFO] 
2020-12-02T11:54:09.5379386Z [INFO] 
2020-12-02T11:54:09.5380391Z [ERROR] Tests run: 19, Failures: 1, Errors: 0, Skipped: 0
```

--- a/tests/integration/src/test/java/org/apache/pulsar/tests/integration/cli/HealthCheckTest.java
+++ b/tests/integration/src/test/java/org/apache/pulsar/tests/integration/cli/HealthCheckTest.java
@@ -108,7 +108,7 @@ public class HealthCheckTest {
     @Test
     public void testBookKeeperDown() throws Exception {
         for (BKContainer b : pulsarCluster.getBookies()) {
-            b.execCmd("pkill", "-STOP", "-f", "BookieServer");
+            b.execCmd("pkill", "-STOP", "-f", "Main");
         }
         assertHealthcheckFailure();
     }

